### ISSUE TITLE
Python 3 support?

### ISSUE THREAD

**Issue by SeanDS** [duncanmmacleod/ligo.org#1]
Great little library! I was wondering if this is intended to work with Python 3, because I'm getting an error. I'd like to include it in a Python 3-only project.

When I try to run the code on Ubuntu 16.04, I get:

```
sean@totoro:~/Workspace/Repositories/ligo.org$ python3
Python 3.5.2 (default, Nov 17 2016, 17:05:23) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from ligo.org import request
>>> response = request("https://dcc.ligo.org/T1100616")
ValueError: PyCapsule_SetPointer called with null pointer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/sean/Workspace/Repositories/ligo.org/ligo/org/ecp.py", line 298, in request
    response = opener.open(request)
  File "/usr/lib/python3.5/urllib/request.py", line 472, in open
    response = meth(req, response)
  File "/usr/lib/python3.5/urllib/request.py", line 582, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.5/urllib/request.py", line 504, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.5/urllib/request.py", line 444, in _call_chain
    result = func(*args)
  File "/home/sean/Workspace/Repositories/ligo.org/ligo/org/ecp.py", line 132, in http_error_401
    self.clean_context()
  File "/home/sean/Workspace/Repositories/ligo.org/ligo/org/ecp.py", line 115, in clean_context
    kerberos.authGSSClientClean(self.context)
SystemError: <built-in function authGSSClientClean> returned a result with an error set
```

In Python 2, it works fine. However, in both Python versions I had to manually install `python3-lxml` (3.5.0-1build1) and `python3-kerberos` (1.1.5-2build1) before I could even successfully import `ligo.org.request`.

**Comment by SeanDS** [comment]
Hang on, the problem was with Kerberos. I had to also install `libkrb5-dev` (via `apt`) in addition to the standard Kerberos Python library (via `pip3 install kerberos`).

Not sure if this is a problem with your package or with one of the upstream dependencies...?

**Comment by duncanmmacleod** [comment]
fixed by #2
